apps/www/components/Article/ReadAloudInline.tsx

```diff
@@ -115,12 +131,15 @@ const SyntheticAudio = ({ meta, t }: { meta: Meta; t: (sting) => string }) => {
             {...colorScheme.set('color', 'text')}
             {...styles.title}
           >
-            {t(`article/${kind}/title`)}
-          </a>{' '}
-          {isSynthetic && (
-            <Editorial.A href='/2022/05/04/helfen-sie-uns-die-synthetische-stimme-zu-verbessern/diskussion'>
-              {t('article/syntheticReadAloud/errorLink')}
-            </Editorial.A>
+            {title}
+          </a>
+          {label && href && (
+            <>
+              {' '}
+              <Editorial.A href={href} {...styles.link}>
+                {label}
+              </Editorial.A>
```

Comment:
Avoid sending css styles to custom elements:
```suggestion
              <Editorial.A href={href}>
                <span {...styles.link}>{label}</span>
              </Editorial.A>
```